google/cloud/language/v1/BUILD.bazel

```diff
@@ -341,10 +341,9 @@ csharp_grpc_library(
 
 csharp_gapic_library(
     name = "language_csharp_gapic",
-    src = ":language_proto_with_info",
-    gapic_yaml = "language_gapic.yaml",
-    package = "google.cloud.language.v1",
-    service_yaml = "//google/cloud/language:language_v1.yaml",
+    srcs = [":language_proto_with_info"],
+    grpc_service_config = "language_grpc_service_config.json",
+    common_resources_config = "@google_cloud_dotnet//:CommonResourcesConfig.json",
```

Comment:
This looks like something which remains constant for all `charp_gapic_library` targets. If this is the case and if `CommonResourcesConfig` resided in `gapic-generator-csharp` (the same repository which defines csharp_gapic_library now), then it could be included by default every time in the `csharp_gapic_library` implementation. This would let us making the surface of the rule simpler (minus one argument) but most importantly remove the dependency on `google-cloud-dotnet` from `googleapis`.